### PR TITLE
first draft for migration-0.10 doc

### PR DESCRIPTION
**Pull request by makeavish**

### CODE REVIEW

File: docs/operate/migration/upgrade-0.10.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- `helm search repo signoz --versions`
+- `helm -n platform upgrade my-release signoz/signoz`
+
+## Steps to run migration script:
+
+### Docker
+
+```bash
+docker run --name signoz_migrate --network clickhouse-setup_default \
+  -it -d signoz/migrate:0.8 -host=clickhouse -port=9000
```
Comment (prashant-shahi) [inline docs/operate/migration/upgrade-0.10.md]: replace `0.8` to `0.10`

File: docs/operate/migration/upgrade-0.10.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+```bash
+docker stop signoz_migrate
+
+docker rm signoz_migrate
+```
+
+### Kubernetes
+
+```bash
+kubectl -n platform run -i -t signoz_migrate --image=signoz/migrate:0.8 \
```
Comment (prashant-shahi) [inline docs/operate/migration/upgrade-0.10.md]: replace `0.8` to `0.10`

File: docs/operate/migration/upgrade-0.10.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+In case of failure and have to run again, make sure to cleanup the container before running the migration script again.
+
+```bash
+docker stop signoz_migrate
+
+docker rm signoz_migrate
+```
+
+### Kubernetes
```
Comment (prashant-shahi) [inline docs/operate/migration/upgrade-0.10.md:44]: Can you include docker swarm instructions as well before K8s one?

Steps will be slightly different in swarm.
- either expose the clickhouse ports and run the similar command via port exposed in host machine
- copy the binary asset inside the clickhouse or query-service container and run the migration script with appropriate arguments

**Review by prashant-shahi — CHANGES_REQUESTED** [review]

**Comment by makeavish** [comment]
@prashant-shahi Please add docker swarm docs and test migration on k8s before merging

**Comment by pranay01** [comment]
merged updated PR  - https://github.com/SigNoz/signoz.io/pull/432
Closing this one

**Pull request closed without merging**